Lib/test/support/bytecode_helper.py

```diff
@@ -58,24 +57,35 @@ def Label(self):
         self.last_label += 1
         return self.last_label
 
-    def complete_insts_info(self, insts):
-        # fill in omitted fields in location, and oparg 0 for ops with no arg.
-        instructions = []
-        for item in insts:
-            if isinstance(item, int):
-                instructions.append(item)
-            else:
-                assert isinstance(item, tuple)
-                inst = list(reversed(item))
-                opcode = dis.opmap[inst.pop()]
-                oparg = inst.pop() if opcode in self.HAS_ARG_OR_TARGET else 0
-                loc = inst + [-1] * (4 - len(inst))
-                instructions.append((opcode, oparg, *loc))
-        return instructions
+    def compareInstructions(self, actual_, expected_):
```

Comment:
Does this compare, or assert that they are equal? `assertInstructionsEqual` perhaps?

Reply:
assertInstructionsMatch perhaps (there is some processing, it's not exactly equality).